src/main/java/com/dabsquared/gitlabjenkins/GitLabPushTrigger.java

```diff
@@ -323,8 +324,19 @@ private GitLabPushCause createGitLabPushCause(GitLabPushRequest req) {
     public RevisionParameterAction createPushRequestRevisionParameter(Job<?, ?> job, GitLabPushRequest req) {
         RevisionParameterAction revision = null;
 
+
+        // add url to remote to work with multiple scm plugin.
```

Comment:
I don't like comments in the code. I get this information also using git blame or the corresponding functionality of an IDE. Comments are bad as they usually aren't touched during refactorings. Which end in comments that tell something about the following code which isn't the truth.
IMO the reason for adding some new code has to be described in the commit message not with a comment within the code.